README.md

```diff
@@ -8,21 +8,29 @@
 You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
 **Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
 
-### Running on the cluster
-1. Install Instances of Custom Resources:
-
+### Running on a KinD cluster
+1. For fedora, you will need the following packages
 ```sh
-kubectl apply -f config/samples/
+sudo dnf install glibc-devel glibc-devel.i686
 ```
-
-2. Build and push your image to the location specified by `IMG`:
-	
+2. Download latest [KinD](https://kind.sigs.k8s.io/docs/user/quick-start) stable version
+3. Install KinD
 ```sh
-make docker-build docker-push IMG=<some-registry>/ingress-node-firewall:tag
+./hack/kind-cluster.sh
 ```
-	
-3. Deploy the controller to the cluster with the image specified by `IMG`:
-
+4. Install custom resource definitions
+```sh
+make install
+```
+6. Build container images
+```sh
+make docker-build IMG=<some-registry>/ingress-node-firewall:tag
+```
+7. Load container image to KinD container(s)
+```sh
+kind load docker-image <some-registry>/ingress-node-firewall:tag
+```
+8. Deploy resources to KinD cluster
 ```sh
 make deploy IMG=<some-registry>/ingress-node-firewall:tag
 ```
```

Comment:
can you add section here in how to run with and w/o webhook with example ?

Reply:
It cannot be disabled currently. Waiting to piggyback on your PR (#38) which defines operator config.

Reply:
Done